OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -104,6 +104,18 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 			dropdown.ShowDropDown("SPAWN_DROPDOWN_TEMPLATE", 150, spawnPoints, setupItem);
 		}
 
+		/// <summary>Splits a string into two parts on the first instance of a given token.</summary>
+		static Pair<string, string> SplitOnFirstToken(string input, string token = "\\n")
+		{
+			if (string.IsNullOrEmpty(input))
```

Comment:
IMO it would be more correct in general to `return Pair("", null)` for "" input. e.g for "A" we return Pair("A", null).